src/pm-conversations/pmConversationsModel.js

```diff
@@ -47,7 +47,7 @@ function keyOfUsers(ids: UserId[], ownUserId: UserId): PmConversationKey {
 }
 
 // Input must indeed be a PM, else throws.
-function keyOfPrivateMessage(msg: Message | Outbox, ownUserId: UserId): PmConversationKey {
+function keyOfPrivateMessage(msg: PmMessage | PmOutbox, ownUserId: UserId): PmConversationKey {
```

Comment:
This comment can get deleted at the same time as the type is updated.

Reply:
Ah, specifically a later commit. So that later change has a hunk like:
```diff
-// Input must indeed be a PM, else throws.
 function keyOfPrivateMessage(msg: PmMessage | PmOutbox, ownUserId: UserId): PmConversationKey {
```

This is fine, but I think it's actually quite legitimate to remove the comment at the time the type gets narrowed to PM-only.

At that point, the comment is talking about a situation that the type says shouldn't exist. In general, if you manage to pass to a function an argument that its type says isn't permitted, the default from the caller's perspective is that anything might happen (much like [undefined behavior](https://en.wikipedia.org/wiki/Undefined_behavior) in language standards), and we don't usually comment on the specifics; if there's a reason to think the caller might need to care, that usually means that either the type or the callsites should be adjusted to bring them into agreement.